packages/api/src/index.js

```diff
@@ -106,6 +108,9 @@ r.add('post', '/api/pins', psa(pinsAdd, RW), [postCors])
 r.add('post', '/api/pins/:requestid', psa(pinsReplace, RW), [postCors])
 r.add('delete', '/api/pins/:requestid', psa(pinsDelete, RW), [postCors])
 
+// Temporary Metaplex upload route, mapped to metaplex user account.
+r.add('post', '/metaplex/upload', withMode(metaplexUpload, RW), [postCors])
+
```

Comment:
Can we move this to just above the `/internal` routes? It's in the middle of the legacy `/api/*` routes right now and I don't want it to accidentally be removed.